tutorials/polkadot-sdk/parachains/build-custom-pallet/build-pallet.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Build the Pallet
+description: Learn how to build a custom pallet for Polkadot SDK-based blockchains with this step-by-step guide. Create and configure a simple counter pallet from scratch.
+---
+
+# Build the Pallet
+
+## Introduction
+
+In Polkadot SDK-based blockchains, runtime functionality is built through modular components called [pallets](/polkadot-protocol/glossary#pallet){target=\_blank}. These pallets are Rust-based runtime modules created using [FRAME (Framework for Runtime Aggregation of Modular Entities)](http://localhost:8000/develop/parachains/customize-parachain/overview/){target=\_blank}, a powerful library that simplifies blockchain development by providing specialized macros and standardized patterns for building blockchain logic.
+A pallet encapsulates a specific set of blockchain functionalities, such as managing token balances, implementing governance mechanisms, or creating custom state transitions.
+
+In this tutorial, you'll create a simple counter pallet with the following features:
+
+- Users can increment and decrement a counter
+- Only a root origin can set an arbitrary counter value
+
+You'll use the [Polkadot SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank}, a pre-configured blockchain template that provides a functional development environment.
+
+After this guide, you'll learn how to create a custom pallet from scratch.
+
+## Prerequisites
+
+To set up your development environment for the Polkadot SDK, you'll need:
+
+- Rust installation - the node template is written in [Rust](https://www.rust-lang.org/){target=\_blank}. Install it by following the [Installation](/develop/parachains/get-started/install-polkadot-sdk){target=\_blank} guide for step-by-step instructions on setting up your development environment
+
+## Download and Compile the Node Template
```

Comment:
This might be subjective, but I think we can rename this title to:
1. avoid naming "node template" directly
2. make it shorter

Wht about: Set Up a Chain Template? Or Sep Up a Template?